components/PostForm/PostForm.js

```diff
@@ -0,0 +1,48 @@
+import styled from "styled-components";
+
+export function Postform() {
+  function handleClick(event) {
+    event.preventDefault();
+    document.getElementById("display").innerHTML =
+      document.getElementById("inputText").value;
+  }
+
+  return (
+    <Post>
+      <div>
+        <label>Post</label>
+        <input type="text" id="inputText" required name="text" />
+        <button type="submit" onClick={handleClick}>
+          Create Post
+        </button>
+        <span id="display"></span>
```

Comment:
Use the state value here:
```suggestion
        <span id="display">{value}</span>
```